webapp/server.js

```diff
@@ -127,27 +142,28 @@ const sessionOptions = {
 
 
 //Need to add remote session storage back in, issues with azure-tables connecting. - Maybe we should switch to elasticache.
-if (env === 'development') {
-  app.use(
-    sessionInMemory(
-      Object.assign(sessionOptions, {
-        name: sessionName,
-        resave: false,
-        saveUninitialized: false,
-      })
-    )
-  );
-} else {
-  app.use(
-    session(
-      Object.assign(sessionOptions, {
-        store: AzureTablesStoreFactory.create(),
-        resave: false,
-        saveUninitialized: false,
-      })
-    )
-  );
-}
+// if (env === 'development') {
```

Comment:
make this use redis always, switches which redis depending on env (localhost for local etc)